### ISSUE TITLE
TypeError after upgrade to Vaadin 14.0.8

### ISSUE THREAD

**Issue by OlliTietavainenVaadin** [vaadin/vaadin-context-menu-flow#110]
Reportedly, this TypeError only appears only after upgrading from Vaadin 14.0.7 to Vaadin 14.0.8
![image](https://user-images.githubusercontent.com/15027701/66477696-564c8380-eaa1-11e9-9e0c-efb7c884d216.png)

**Comment by pekam** [comment]
Closing as a duplicate of #108
